CredentialProvider/Assets.xcassets/clear-icon.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "combinedShape.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "combinedShape@2x.png",
```

Comment:
nit: Could we replace these pngs with a templated pdf?

Do we have a clear icon we could re-use?

Reply:
Same comment for the rest of the png files

Reply:
I don't know if we have those since these were copied from Lockwise. What do you think @rilury ? Did you see PDFs for any of the assets anywhere in the Lockwise project?

Reply:
Indeed, the pngs were copied from Lockwise and  the assets in that project were in png format. I did not find PDFs for any of those. We can change them sure, but I think we need help  from someone to convert them to PDFs.